app/design/frontend/default/models/template/test/test.phtml

```diff
@@ -1,188 +1,48 @@
+   <th>CITY</th>
+   <th>PINCODE</th>
+   <th>CONTACT</th>
    <th>Delete</th>
   </tr>
  <?php
- $model = Mage::getModel('test/test')->getCollection();
- foreach($model as $data)
-  {
-    echo '<tr>';
-    echo '<td>'.$data->getTestId().'</td>';
-    echo '<td>'.$data->getTitle().'</td>';
-    echo '<td>'.$data->getFilename().'</td>';
-    echo '<td>'.$data->getContent().'</td>';
-    echo '<td>'.$data->getStatus().'</td>';
-    echo '<td>'.$data->getCreatedTime().'</td>';
-    echo '<td>'.$data->getUpdateTime().'</td>';
-    ?>
-     <!--  <td><a class="edit" style="cursor:pointer;">Edit</a></td> -->
-      <td><a href='?id=<?php echo $data->getTestId() ?>'>Delete</a></td>
-    <?php
-     echo '</tr>';
-   }
- ?>
- </table>
- </div>
-
- <!--===================================Added second table here===================================-->
- <div id="right-div">
- <h3 id="heading2"><center>Enter all fields to Add data in second table</center></h3>
- <form role="form" action="<?php echo $this->getUrl('test/user_view/save2'); ?>" method="post">
-  <div id="form_div">
-   <label id="test_id_label2" style="display:none;">Test_Id</label>
-   <input id="test_id2" type="text" name="testid" style="display:none;" />
-   <br /><br />
-   <label>Title</label>
-   <input id="title2" type="text" name="title" />
-   <br /><br />
-    <label>File Name</label>
-   <input id="file_name2" type="text" name="filename" />
-   <br /><br />
-   <label>Content</label>
-   <input id="content2" type="text" name="content" />
-   <br /><br />
-   <label>Status</label>
-   <input id="status2" type="text" name="status" />
-   <br /><br />
-   <input id="add_button2" type="submit" value="Add" disabled="disabled" />
-   <input type="button" class="edit2" value="Edit" style="cursor:pointer;" />
-  </div>
-  <br /><br />
- </form>
- <table border="">
-  <tr>
-   <th>Test_Id</th>
-   <th>Title</th>
-   <th>File Name</th>
-   <th>Content</th>
-   <th>Status</th>
-   <th>Created Time</th>
-   <th>Updated Time</th>
-   <!-- <th>Edit</th> -->
-   <th>Delete</th>
-  </tr>
- <?php
- $model = Mage::getModel('test/test2')->getCollection();
- foreach($model as $data)
-  {
-    echo '<tr>';
-    echo '<td>'.$data->getTestId().'</td>';
-    echo '<td>'.$data->getTitle().'</td>';
-    echo '<td>'.$data->getFilename().'</td>';
-    echo '<td>'.$data->getContent().'</td>';
-    echo '<td>'.$data->getStatus().'</td>';
-    echo '<td>'.$data->getCreatedTime().'</td>';
-    echo '<td>'.$data->getUpdateTime().'</td>';
-    ?>
-     <!--  <td><a class="edit" style="cursor:pointer;">Edit</a></td> -->
-      <td><a href='?id2=<?php echo $data->getTestId() ?>'>Delete</a></td>
-    <?php
-     echo '</tr>';
-   }
- ?>
- </table>
- </div>
- <!--=============================================================================================-->
-
-</div>
- <script type="text/javascript">
-  $("input[type='text']").on("keyup", function()
+  $user_model = Mage::getModel('user/user')->getCollection();
```

Comment:
this database function should go in block class. never we shold write db code in phtml